magicblock-config/tests/fixtures/05_all-goes.toml

```diff
@@ -10,4 +10,4 @@ lifecycle = "replica"
 sigverify = false
 
 [ledger]
-reset = false
+resume-strategy = "replay-and-resume"
```

Comment:
Again I wasn't very good with suggestions for the names. _replay_ kinda implies _resume_ right?

So I'd think the enum values should be: `reset | replay | resume-only`.

`resume-only` could also be just `resume` - up to you. The `only` part just makes it double clear that we won't replay.